components/Navigation/index.js

```diff
@@ -18,57 +24,59 @@ export default function Navigation() {
   return (
     <nav>
       <StyledNavList>
-        <StyledNavigationItem>
+        <StyledNavigationItem
+          className="navTransition"
```

Comment:
Is there a reason why you use ``className`` in a styled-component?

Reply:
No, I removed them :)